Makefile

```
@@ -9,3 +9,6 @@ npm_install:
 
 bundle_install:
 	bundle install
+
+start:
+	nanoc view
```

Additionally there's `nanoc live` which does live reloading when things change. I used `view` when doing template work as the live reloader went a bit nuts refreshing for 900 pages, but in everyday work it's probably what people want.

```suggestion
start:
	nanoc view
	
watch:
  bundle exec nanoc live
```